src/cli/commands/config.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright 2022-2023 Protocol Labs
+// SPDX-License-Identifier: MIT
+//! This mod triggers a config reload in the IPC-Agent Json RPC server.
+
+use async_trait::async_trait;
+use clap::Args;
+use std::fmt::Debug;
+
+use crate::cli::commands::get_ipc_agent_url;
+use crate::cli::{CommandLineHandler, GlobalArguments};
+use crate::config::json_rpc_methods;
+use crate::jsonrpc::{JsonRpcClient, JsonRpcClientImpl};
+use crate::server::ReloadConfigParams;
+
+/// The command to create a new subnet actor.
+pub(crate) struct ReloadConfig;
+
+#[async_trait]
+impl CommandLineHandler for ReloadConfig {
+    type Arguments = ReloadConfigArgs;
+
+    async fn handle(global: &GlobalArguments, arguments: &Self::Arguments) -> anyhow::Result<()> {
+        log::debug!("reload config with args: {:?}", arguments);
+
+        let params = ReloadConfigParams {
+            path: arguments.path.clone(),
```

Comment:
I am wondering if we should pass the full config instead of a path to the json rpc method for config reload, what if the cli and the jsonrpc server live in different machines?

Reply:
I could be wrong, but I have this feeling that config reload is a really privileged operation, we should put some restrictions as not to expose this method to the network, who know what ppl might do. Probably only same machine invocation has fewer risks? If we wan to pass the Config struct, it's setting the configs and we should persists those updated config to survive server restarts, probably it's not called a config reload and we can handle it in another method?

Reply:
>  If we wan to pass the Config struct, it's setting the configs and we should persists those updated config to survive server restarts, probably it's not called a config reload and we can handle it in another method?

This is a good point. 

> I could be wrong, but I have this feeling that config reload is a really privileged operation, we should put some restrictions as not to expose this method to the network, who know what ppl might d

I am not that worried about this at this point because we need to figure out authentication and access rights. For now we are assuming that for M2 users won't expose their agent publicly. 

I guess for simplicity we can leave it like this for now, but do you mind tracking this in an issue so we tackle it in the future?

Reply:
Added, see #88.